client/components/login/login.js

```diff
@@ -59,8 +60,8 @@ export default class Login extends React.Component {
   }
 
   getEmailField = (input_fields) => {
-    const {email, errors} = this.state;
-    const {language} = this.props;
+    const { email, errors } = this.state;
+    const { language } = this.props;
```

Comment:
please avoid these kind of style changes. If you want to add an automatic formatting tool that does this kind of change, please open a new issue and a new PR just for that.

This applies to any change in this PR which is not directly related to the feature you're working on.